sql/core/src/main/scala/org/apache/spark/sql/execution/python/streaming/TransformWithStateInPySparkExec.scala

```diff
@@ -494,7 +521,7 @@ object TransformWithStateInPandasExec {
       stateStoreCkptIds = None
     )
 
-    new TransformWithStateInPandasExec(
+    new TransformWithStateInPySparkExec(
```

Comment:
Are we changing the name of the physical operator also for the current pandas tws operator ?

Reply:
Yes, but we retain the "shortName" as it is for Pandas path, so wouldn't have a compatibility issue.